frontend/common/src/components/UserProfile/SkillAccordion/SkillAccordion.tsx

```diff
@@ -33,6 +33,29 @@ export interface SkillAccordionProps {
   headingLevel?: HeadingLevel;
 }
 
+/**
+ * @param skillsArray - array of skills attached to experience
+ * @param accordionSkill - single skill that is the focus of a skill accordion
+ * @returns justification for the skill that matches the accordion skill or return an empty string
+ */
+const grabSkillJustification = (
+  skillsArray: Skill[],
+  accordionSkill: Skill,
+): string => {
+  // find the skill from the array that matches the accordion skill, this results in an array of one skill
+  const specificSkill = skillsArray.filter(
```

Comment:
If you're looking for a single item in the array, you should probably use [Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of Array.filter

Reply:
like so https://github.com/GCTC-NTGC/gc-digital-talent/pull/5308/commits/9c578f2f801d1dffbdec0c32d8879fcc1da1d912 ?

Reply:
perfect!